NBitcoin/BIP32/KeyPath.cs

```diff
@@ -40,6 +40,23 @@ public KeyPath(string path)
 				.ToArray();
 		}
 
+		public static KeyPath FromBytes(byte[] data)
+		{
+			if (data.Length % 4 != 0)
+				throw new ArgumentOutOfRangeException("data length is not suited for KeyPath");
+			var depth = data.Length / 4;
+			uint[] result = new uint[depth];
+			for (int i = 0;  i < depth; i++)
+			{
+				result[i] = BitConverter.ToUInt32(data, i * 4);
+			}
+
+			return new KeyPath(result);
+		}
+
+		public byte[] ToBytes() =>
+			Indexes.Select(i => BitConverter.GetBytes(i)).Aggregate((a, b) => a.Concat(b)).ToArray();

```

Comment:
This is okay because the `Indexes` has just a couple of elements but for collections with more elements it could be good to use `SpanExtensions .AsBytes()`, `CopyTo`, or `Cast/TryCast` (I don't know if these last methods are still available)

Reply:
Sorry for my ignorance but what do you mean by `SpanExtensions` ? I could not found anything besides [internal module of Kestrel](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.server.kestrel.internal.system.spanextensions.asbytes?view=aspnetcore-2.0)

Reply:
Meh yes that's fine span does not work on some target, let's keep it like this now.

Reply:
netstandard2.0